nifi-nar-bundles/nifi-standard-bundle/nifi-standard-processors/src/main/java/org/apache/nifi/processors/standard/CompressContent.java

```diff
@@ -124,7 +126,7 @@
         .build();
     public static final PropertyDescriptor MODE = new PropertyDescriptor.Builder()
     .name("Mode")
-    .description("Indicates whether the processor should compress content or decompress content. Must be either 'compress' or 'decompress'")
+    .description("Indicates whether the processor should compress content or decompress content. Must be either 'compress' or 'decompress'. Data that is compressed with Snappy Hadoop can not be decomressed using this processor.")
```

Comment:
Instead of updating the description, could we add a customValidate method to make the processor invalid in case both SNAPPY HADOOP is selected for the compression format and DECOMPRESS is selected for the mode?

Reply:
done